example/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: built_html_example
+version: 1.0.0
+description: Example to build *.html files from *.template.html source.
+
+environment:
+  sdk: '>=2.0.0 <3.0.0'
+
+dependencies:
```

Comment:
nit: we should be able to omit this

Reply:
What do you mean by that?

Reply:
Since there are no dependencies here we should be able to leave the "dependencies" section out completely